packages/babel-preset-flow/src/normalize-options.js

```diff
@@ -0,0 +1,25 @@
+import { OptionValidator } from "@babel/helper-validator-option";
+const v = new OptionValidator("@babel/preset-flow");
+
+export default function normalizeOptions(options = {}) {
+  let { all } = options;
+  const { allowDeclareFields } = options;
```

Comment:
nbd, maybe just do one var declaration?

Reply:
ESLint will complain, because one has `let` and one has `const`.